src/app/community-view/community-view.component.spec.ts

```diff
@@ -59,6 +61,23 @@ const community: CommunityResponse = {
   ]
 };
 
+const currentUser = {
+  id: authenticatedUser.userId,
+  userName: authenticatedUser.userName,
+  firstName: authenticatedUser.firstName,
+  lastName: authenticatedUser.lastName,
+  email: authenticatedUser.email,
+  coach: false,
+};
+
+const communityUserRegistrations: CommunityUserRegistrationResponse[] = [
+  {
+    user: currentUser,
+    approvedByUser: true,
+    approvedByCommunity: true
```

Comment:
Isn't it better to set `approvedByCommunity` to `null` here? The authenticated user sends a request to join closed community and that means the request is not approved by the community yet.

Reply:
thank you. done